templates/volontulo/organization_form.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
```

Comment:
OK, but with #73 there is new 2 additional step: extending col1/col2 and those extends base.